modules/monitoring-configuring-persistent-storage.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: CONCEPT
+[id="configuring_persistent_storage_{context}"]
+= Configuring persistent storage
+
+Running cluster monitoring with persistent storage means that your metrics are stored to a persistent volume (PV) and can survive a pod being restarted or recreated. This is ideal if you require your metrics or alerting data to be guarded from data loss. For production environments, it is highly recommended to configure persistent storage. Because of the high IO demands, it is advantageous to use local storage.
```

Comment:
`This` is a little vague for a subject of a sentence. What do you think of this rephrase? 

`This PV storage is ideal if you require your metrics or alerting data...`

Reply:
Those are good points, but this content is moved and not new, and so changing it is outside the scope of this PR. Leaving as-is